Firestore/Example/Firestore.xcodeproj/project.pbxproj

```diff
@@ -709,6 +743,7 @@
 				6003F5B5195388D20070C39A /* Tests */,
 				54764FAC1FAA0C390085E60A /* CoreTests */,
 				54C9EDF22040E16300A969CD /* SwiftTests */,
+				5CAE131A20FFFED600BE9A4A /* Firestore_Benchmarks_iOS */,
```

Comment:
Could rename this in the left-hand nav to just "Benchmarks"?

Reply:
Done.